docs/database/hbase.md

```diff
@@ -17,9 +17,12 @@ described on this page.
 <!-- semconv db.hbase(full,tag=tech-specific) -->
 | Attribute  | Type | Description  | Examples  | [Requirement Level](https://opentelemetry.io/docs/specs/semconv/general/attribute-requirement-level/) | Stability |
 |---|---|---|---|---|---|
-| [`db.name`](../attributes-registry/db.md) | string | The HBase namespace. [1] | `mynamespace` | `Conditionally Required` If applicable. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| [`db.collection.name`](../attributes-registry/db.md) | string | The HBase table name. [1] | `mytable`; `ns:table` | `Conditionally Required` If applicable. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| [`db.collection.namespace`](../attributes-registry/db.md) | string | The HBase namespace. [2] | `mynamespace` | `Conditionally Required` If applicable and is not default. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
 
-**[1]:** For HBase the `db.name` should be set to the HBase namespace.
+**[1]:** If table name includes the namespace, the `db.collection.name` SHOULD be set to the full table name.
+
+**[2]:** When performing table-related operations, the instrumentations SHOULD extract the namespace from the table name according to the [HBase table naming conventions](https://hbase.apache.org/book.html#namespace_creation).
```

Comment:
HBase tables can be named as `table` (default namespace) or `namespace:table`. 

Namespace is more than a naming convention e.g. it affects permissions or you can list tables within database.

However common operations (such as get/add rows) don't have namespace as a dedicated parameter, it's provided only as a part of a table name. See  https://hbase.apache.org/book.html#_using_rest_endpoints for the deatils..

Given it's easy to extract and there is no other way to obtain it, I added this note.

If we don't extract it, we won't have namespace as a separate attribute for common HBase operations. 

We'll also have to update how span name is constructed for HBase - it should include full table name and not the namespace.